chalice/app.py

```diff
@@ -418,12 +418,12 @@ class Chalice(object):
 
     FORMAT_STRING = '%(name)s - %(levelname)s - %(message)s'
 
-    def __init__(self, app_name, configure_logs=True, env=None):
+    def __init__(self, app_name, debug=False, configure_logs=True, env=None):
```

Comment:
My preference given we're pre GA is to order them based on frequent usage and I would expect debug to be more common than the other options.  So I vote to keep it as is.